docs/user-guide/global-configurations/approval-policy.md

```diff
@@ -0,0 +1,136 @@
+# Approval Policy
+
+## Introduction [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
+
+When it comes to critical environments (let's say, production), you as a superadmin might want to introduce an approval flow for application deployment or changes made to the configuration files. Enforcing such restrictions will prevent unwanted deployments and direct modifications to sensitive configurations.
+
+The **Approval Policy** feature in Devtron lets you introduce an approval mechanism whenever your users perform the following actions:
+
+- Deploying an Application
+- Changes in Deployment Template
+- Changes in ConfigMap
+- Changes in Secret
+
+![Figure 1a: Approval for Deployment](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/approval-policy/deployment-approval.gif)
+
+![Figure 1b: Approval for Configuration Change](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/approval-policy/config-approval.gif)
+
+---
+
+## Creating an Approval Policy
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permissions to create an approval policy.
+{% endhint %}
+
+1. Go to **Global Configurations** → **Approval Policy**.
+
+    ![Figure 2: Approval Policy](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/approval-policy/gc-approval-policy.jpg)
+
+2. Click **+ Create Profile**.
+
+    ![Figure 3: 'Create Profile' Button](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/approval-policy/create-approval-profile.jpg)
+
+3. Give a name to the policy, e.g., *`banking-prod-approval`*, and add a description (optional) preferably explaining what it does.
+
+    ![Figure 4: Entering Policy Details](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/approval-policy/policy-desc.jpg)
+
+4. Choose the mandatory number of approvals your users must get to proceed with a deployment/configuration change. The permissible limit ranges from one approval (minimum) to six approvals (maximum). Additionally, you can decide who can grant approval from the following 3 options:
```

Comment:
this possible range is only valid for any approver case, with specific I can achieve higher limits